client/blocks/inline-help/inline-help-search-card.jsx

```diff
@@ -139,9 +158,9 @@ const mapStateToProps = ( state, ownProps ) => ( {
 } );
 const mapDispatchToProps = {
 	track: recordTracksEvent,
-	setInlineHelpSearchQuery,
-	selectNextResult,
-	selectPreviousResult,
+	seSearchQuery: setInlineHelpSearchQuery,
```

Comment:
Should this be `setSearchQuery`? Probably missing a `t`

Reply:
Looks like, @cpapazoglou spotted it as well and just pushed a change 👌

Reply:
outch